galaxy_ng/tests/integration/dab/test_dab_rbac_contract.py

```diff
@@ -204,16 +208,26 @@ def assert_assignments(gc, user, namespace, expected=0):
             assert related[field].startswith(GALAXY_API_PATH_PREFIX)
 
 
-@pytest.mark.parametrize("by_api", ["dab", "pulp"])
-def test_give_custom_role_object(
-    request, galaxy_client, custom_role_creator, namespace, by_api, random_username
-):
-    data = NS_FIXTURE_DATA.copy()
-    data["name"] = "galaxy.namespace_custom_object_role"
-    data["content_type"] = "galaxy.namespace"
-    custom_obj_role = custom_role_creator(data)
-
+@pytest.mark.parametrize("by_role_api", ["dab", "pulp"])
+@pytest.mark.parametrize("by_assignment_api", ["dab", "pulp"])
+def test_give_custom_role_object(galaxy_client, custom_role_creator, namespace, by_role_api, by_assignment_api):
     gc = galaxy_client("admin")
+
+    if by_role_api == 'pulp' and by_assignment_api == 'dab':
+        pytest.skip('This is not supported, the compatbility shim is only for the pulp assignment API')
```

Comment:
This is saying - you can't create a role in the pulp API and then make an object assignment in the DAB API with that.

However, the dab-pulp combination does work. If people don't like the double-parameterization, I could just change it to only test dab-dab, and pulp-pulp, which makes more sense. You should create the role and do the assignment in the same API.